### PR TITLE
document k8s support

### PR DESCRIPTION
**Pull request by AviAvni**

### CODE REVIEW

File: k8s_support.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+### Prerequisites
+
+Before you begin, make sure you have Helm installed on your Kubernetes cluster.
+
+To deploy FalkorDB to Kubernetes we need to use:
+
+* [Helm charts](https://bitnami.com/stack/redis/helm)
+* [Docker image](https://hub.docker.com/r/falkordb/falkordb)
+
+And follow this steps:
```
Comment (swilly22) [inline k8s_support.md:20]: follow these steps

File: k8s_support.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+### Step 1: Create a `values.yaml` File
+
+1.Create a values.yaml file with the following content:
+
+```yaml
+image:
+  registry: docker.io
+  repository: falkordb/falkordb
+  tag: "4.0"
```
Comment (swilly22) [inline k8s_support.md:30]: There's no way we will remember to update this every time we release a new version

File: k8s_support.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+master:
+  extraFlags:
+  - "--loadmodule /FalkorDB/bin/linux-x64-release/src/falkordb.so"
+
+slave:
+  extraFlags:
+  - "--loadmodule /FalkorDB/bin/linux-x64-release/src/falkordb.so"
```
Comment (swilly22) [inline k8s_support.md:38]: Is this really necessary, our docker does that by default

File: k8s_support.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+```
+This file specify the FalkorDB image(you can choose different tags)
+and configure the master and slave to load the FalkorDB module.
+For additional configurations [see the official Helm chart documentation](https://github.com/bitnami/charts/blob/main/bitnami/redis/values.yaml)
```
Comment (swilly22) [inline k8s_support.md:42]: This is a bit confusing, we ask the reader to use `falkordb` docker image but point them to read configuration from bitnami Redis.

File: k8s_support.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+### Step 2: Install FalkorDB Helm Charts
+Install the helm charts using the following command:
+
+```bash
+helm install -f values.yaml my-falkordb oci://registry-1.docker.io/bitnamicharts/redis
```
Comment (swilly22) [inline k8s_support.md:48]: Should we introduce `oci://registry-1.docker.io/bitnamicharts/falkordb` ?

File: k8s_support.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+This command deploys FalkorDB with the configuration from values.yaml.
+After running this command, instructions on how to connect to the FalkorDB server will be displayed.
+
+### Step 3: Retrieve the FalkorDB Password
+
+To connect to FalkorDB, you need the Redis password. Retrieve it using the following command:
+
+```bash
+export REDIS_PASSWORD=$(kubectl get secret --namespace default my-release-redis -o jsonpath="{.data.redis-password}" | base64 -d)
+```
```
Comment (swilly22) [inline k8s_support.md:60]: I'll assume that if we run our own Helm chart we'll have full control over which password is used.

File: k8s_support.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+    6) 1) (integer) 6
+    7) 1) (integer) 7
+    8) 1) (integer) 8
+    9) 1) (integer) 9
+   10) 1) (integer) 10
+3) 1) "Cached execution: 0"
+   2) "Query internal execution time: 0.290600 milliseconds"
+```
+
+In summary, this guide provides clear steps for deploying FalkorDB on Kubernetes using Helm charts and Docker images. It covers the creation of a values.yaml file for configuration, Helm chart installation, password retrieval, enabling external connections, connecting to FalkorDB with redis-cli, and running a basic Cypher query for verification. We hope this documentation helps you successfully set up FalkorDB in your Kubernetes environment. If you have any questions or encounter any issues during the process, please don't hesitate to reach out for assistance. Thank you for choosing FalkorDB!
```
Comment (swilly22) [inline k8s_support.md:105]: "clear steps" -> "steps"
"helps you successfully set up" -> "helped you to set up"

**Review by swilly22 — CHANGES_REQUESTED** [review]

File: k8s_support.md (hunk 2 above)
Comment (AviAvni) [reply k8s_support.md:30]: it is example

File: k8s_support.md (hunk 3 above)
Comment (AviAvni) [reply k8s_support.md:38]: the charts override the starting point of the docker image

File: k8s_support.md (hunk 4 above)
Comment (AviAvni) [reply k8s_support.md:42]: we hosted on redis

File: k8s_support.md (hunk 6 above)
Comment (AviAvni) [reply k8s_support.md:60]: there is a password parameter in this charts

File: k8s_support.md (hunk 7 above)
Comment (AviAvni) [reply k8s_support.md]: interesting this is the ChatGPT text but made the change

File: k8s_support.md (hunk 3 above)
Comment (swilly22) [reply k8s_support.md:38]: Should we publish our own chart ?

File: k8s_support.md (hunk 4 above)
Comment (swilly22) [reply k8s_support.md:42]: Right but can we create our own chart which inherits / build on top of Redis ?

File: k8s_support.md (hunk 6 above)
Comment (swilly22) [reply k8s_support.md:60]: Right, I'll assume we can have full control over the password if we had our own chart

**Pull request merged**